src/frontend/src/components/recoveryMethodsSection.ts

```diff
@@ -1,6 +1,85 @@
 import { TemplateResult, html } from "lit-html";
 import { deviceListItem, Device } from "./deviceListItem";
 
+const recoveryByType = ({
+  type,
+  recoveries,
+}: {
+  type: "phrase" | "device";
+  recoveries: Device[];
+}) => recoveries.filter((device) => device.recovery === type);
+
+const sectionList = ({
+  recoveries,
+  onAddRecovery,
+}: {
+  recoveries: Device[];
+  onAddRecovery?: () => void;
+}): TemplateResult => {
+  console.log(recoveries);
+  const recoveryPhrases = recoveryByType({ type: "phrase", recoveries });
```

Comment:
I guess per doesn't matter much but if it would, you can probably do this in one shot with a reducer. Something like:

```
const {phrases, devices} = recovries.reduce((acc, device) => {
  if (device.recovery === "phrase") {
     return {phrases: [...acc.phrases], devices: [...acc.devices, device]}
  }

     return {phrases: [...acc.phrases, device], devices: [...acc.devices]}  
}, {phases: [], devices []})
```

Just sharing in case you like it the idea.